### ISSUE TITLE
Opening the mapping stats crashes Enigma

### ISSUE THREAD

**Issue by jordanamr** [FabricMC/Enigma#503]
When opening a jar, then an Enigma mappings folder and trying to view the mappings stats, it shows this instead :

```
java.lang.NullPointerException: Cannot invoke "String.startsWith(String)" because the return value of "cuchaz.enigma.translation.representation.entry.ClassEntry.getPackageName()" is null
	at cuchaz.enigma.gui.stats.StatsGenerator.generate(StatsGenerator.java:105)
	at cuchaz.enigma.gui.dialog.StatsDialog.lambda$show$1(StatsDialog.java:37)
	at cuchaz.enigma.gui.dialog.ProgressDialog.lambda$runOffThread$1(ProgressDialog.java:97)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

Enigma version: 2.3.1-all.jar from the repository.
OS: Arch
Java version: Installed from the jdk17-openjdk AUR 
```
openjdk version "17.0.7" 2023-04-18
OpenJDK Runtime Environment (build 17.0.7+7)
OpenJDK 64-Bit Server VM (build 17.0.7+7, mixed mode)
```

**Comment by jordanamr** [comment]
FYI is also happens while simply opening the jar and trying to see stats without any mappings opened.
I'm providing the jar in question in a zip file
[core.zip](https://github.com/FabricMC/Enigma/files/11439593/core.zip)

**Comment by mudkipdev** [comment]
The null pointer exception still occurs, only on line 70 this time. I attempted to fix it in #504.
